joomla/joomla-cms
# [4.0] Remove unused file

### Summary of Changes

In #7003, the `default_batch` templates were replaced by `default_batch_body` and `default_batch_footer`. The `default_batch` templates have been marked as deprecated from 3.4 and deleted in #11762 - all but one. This PR deletes the last `default_batch` template which has been unused since 3.4 (except maybe in some 3rd party template overrides), to avoid confusion and duplicated code.

The alternative would be to mark it as deprecated and remove it later.


### Testing Instructions
Apply the PR and make sure that batch processing still works for contacts. Search the code for occurrences of '"default_batch" or `loadTemplate('batch')` and make sure that the template is not used.
